apps/test/unit/netsim/NetSimLobby.js

```diff
@@ -0,0 +1,51 @@
+import sinon from 'sinon';
```

Comment:
Even though the other netsim test files don't follow this convention, it'd be good to start using it:
Can you rename NetSimLobby.js to NetSimLobbyTest.js?

Reply:
Definitely!